README.md

```diff
@@ -4,37 +4,45 @@
 #### Table of Contents
 
 1. [Description](#description)
-2. [Setup - The basics of getting started with p9_instance_setup](#setup)
-    * [What p9_instance_setup affects](#what-p9_instance_setup-affects)
-    * [Beginning with p9_instance_setup](#beginning-with-p9_instance_setup)
-3. [Usage - Configuration options and additional functionality](#usage)
+1. [Limitations](#limitations)
+1. [What p9_instance_setup affects](#what-p9_instance_setup-affects)
+1. [Usage - Configuration options and additional functionality](#usage)
+    1. [Via Puppet](#via-puppet)
+    1. [Via Bolt Plans](#via-bolt-plan)
+        1. [Setup](#setup)
+        1. [Plan Examples](#plan-examples)
+            1. [Parameters](#Parameters)
+            1. [Setup_instances](#setup_instances)
+            1. [Manage_software](#manage_software)
+            1. [Setup_user](#setup_user)
+            1. [Upload_auth_files](#upload_auth_files)
+            1. [Upload_user_files](#upload_user_files)
 
 ## Description
 
-This module can configure a platform9 instance with the needed software to be able to run puppet perf simulations.
+This module can configure a platform9 instance with the needed software to be able to run 
+puppet perf simulations. It will also create and setup user(s) to be used for the same.
 
-## Setup
+## Limitations
 
-### What p9_instance_setup affects
+This modile is only able to configue centos instances.
```

Comment:
Centos should be capitalized here for consistency with the 'Via Bolt Plan' section below.